vividus/src/main/resources/org/vividus/spring-vividus.xml

```diff
@@ -76,4 +76,6 @@
 
     <util:list id="storyReporters" />
 
+    <bean id="fluentEnumConverter" class="org.jbehave.core.steps.ParameterConverters.FluentEnumConverter" />
```

Comment:
is `id` really needed?

Reply:
yes, because then found bean from test-spring.xml file

Reply:
which stage is affected by `test-spring.xml`?

Reply:
During context initialization: `Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'org.jbehave.core.steps.ParameterConverters$FluentEnumConverter' available: expected single matching bean but found 3: fluentTrimmedEnumConverter,org.jbehave.core.steps.ParameterConverters.FluentEnumConverter#0,org.jbehave.core.steps.ParameterConverters.FluentEnumConverter#1`